src/quo2/components/avatars/icon_avatar.cljs

```diff
@@ -22,17 +22,21 @@
   (let [{component-size :component icon-size :icon} (get sizes size)
         circle-color                                (colors/resolve-color color theme opacity)
         icon-color                                  (colors/resolve-color color theme)]
-    [rn/view
-     {:style {:width            component-size
-              :height           component-size
-              :border-radius    component-size
-              :border-width     (when border? 1)
-              :border-color     (colors/theme-colors colors/neutral-20 colors/neutral-80 theme)
-              :background-color circle-color
-              :justify-content  :center
-              :align-items      :center}}
-     [icons/icon icon
-      {:size  icon-size
-       :color icon-color}]]))
+    (if (= (type icon) Keyword)
```

Comment:
Can use keyword?
```clj
(keyword? :x)
;;=> true
(keyword? true)
;;=> false
```

Reply:
perhaps we can be even stricter and check for icon keywords, i.e `:i/*` 🤔

Reply:
@J-Son89 And what if it is not an icon keyword? I am basically checking if it's a keyword to return `icon` or otherwise return an `image`